### PR TITLE
Document PackedVector4Array

### PR DESCRIPTION
**Pull request by Calinou**
This type was added in Godot 4.3.

- See https://github.com/godotengine/godot-docs-user-notes/discussions/61#discussioncomment-10492602.

### CODE REVIEW

File: tutorials/scripting/gdscript/gdscript_basics.rst [hunk 1]
```diff
@@ -895,6 +895,7 @@ arrays. They are therefore only recommended to use for large data sets:
 - :ref:`PackedStringArray <class_PackedStringArray>`: An array of strings.
 - :ref:`PackedVector2Array <class_PackedVector2Array>`: An array of :ref:`Vector2 <class_Vector2>` values.
 - :ref:`PackedVector3Array <class_PackedVector3Array>`: An array of :ref:`Vector3 <class_Vector3>` values.
+- :ref:`PackedVector3Array <class_PackedVector4Array>`: An array of :ref:`Vector3 <class_Vector4>` values.
```
Comment (raulsntos) [inline tutorials/scripting/gdscript/gdscript_basics.rst:898]:
```suggestion
- :ref:`PackedVector3Array <class_PackedVector4Array>`: An array of :ref:`Vector4 <class_Vector4>` values.
```

File: _extensions/gdscript.py [hunk 2]
```diff
@@ -358,6 +358,7 @@ def innerstring_rules(ttype):
                         "PackedVector2iArray",
                         "PackedVector3Array",
                         "PackedVector3iArray",
```
Comment (raulsntos) [inline _extensions/gdscript.py:360]: A bit off-topic but there's no `PackedVector2iArray` or `PackedVector3iArray`.

**Review by raulsntos — APPROVED** [review]

**Comment by raulsntos** [comment]
Since the type was added in 4.3, shouldn't we add the `cherrypick:4.3` label?

**Comment by mhilbrunner** [comment]
Thanks!

**Comment by mhilbrunner** [comment]
Cherry-picked to 4.3 in https://github.com/godotengine/godot-docs/pull/10038.

**Pull request merged**